_contentTemplates/common/filtering.md

```diff
@@ -5,6 +5,7 @@
 | Parameter | Type and Default Value | Description |
 | --- | --- | --- |
 | `DefaultFilterOperator` | `FilterOperator` | Sets the default filter operator in the column it is declared for. Accepts a member of the `FilterOperator` enum. The selected operator must be applicable for the specific data type. Check the supported options in the [Filter Operators]({%slug common-features-filter-operators%}) article.
+| `FilterOperators` | `List<FilterListOperator>` | Specifies the available operators. Must contain only [supported filter operators]({%slug common-features-filter-operators%}) for the specific data type. If not defined, a default list of available operators will be used based on the field type.
```

Comment:
"a default list of available operators will be used based on the field type." - consider active voice, e.g. "the Grid will use/show a default list..."